cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -1397,3 +1506,42 @@ func CleanSriov() {
 	}
 	sriov.WaitStable(sriovclient)
 }
+
+func getDeviceMac(pod *corev1.Pod, device string) (string, error) {
```

Comment:
no need for this function lets just static allocate the mac addresses

Reply:
Done